packages/copyable/src/Copyable.tsx

```diff
@@ -70,12 +70,14 @@ const codeStyle = css`
   display: inline-flex;
   align-items: center;
   height: 100%;
-  width: 100%;
+  // Button is 75px wide. Need to set a smaller width so that we're able to scroll when text overflows.
+  width: calc(100% - 75px);
   padding-left: 12px;
   border: 1px solid;
   border-radius: 4px;
   font-size: 14px;
-  overflow: hidden;
+  overflow-x: auto;
+  overflow-y: hidden;
```

Comment:
theoretically it shouldn't be possible to overflow in the y direction since the child must be a single string